src/cascadia/WindowsTerminalUniversal/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+
+#include "Generated Files\winrt\TerminalApp.h"
```

Comment:
nit: I think the preferred style is to have "Generated Files" on the include line, so that all these headers can just be included as `#include <winrt/TerminalApp.h>`. Not worth blocking over however.

Reply:
I am having a hard time finding another instance of what you're asking elsewhere in our solution.

I found CPP files set to compile with $(GeneratedFilesDir) and I found one cross-project reference to generated files added to the additional includes. But I didn't find any project files that added their own Generated Files dir to the includes line.

Reply:
OK. Trying to make this implicit causes conflicts between the import reference from the other project and the post-merged generation files. For whatever reason, it works as is.